<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionShowBox"/>
    <addaction name="actionShowAxis"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>&amp;Style</string>
    </property>
    <addaction name="actionCPKSpheres"/>
    <addaction name="actionBallsAndSticks"/>
    <addaction name="actionSticks"/>
    <addaction name="actionWires"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStyle"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionShowBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show bounding box</string>
   </property>
  </action>
  <action name="actionShowAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show axis</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About VisMol...</string>
   </property>
   <property name="toolTip">
    <string>Acerca de Visualizador de Mol√©culas</string>
   </property>
  </action>
  <action name="actionBallsAndSticks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Balls and sticks</string>
   </property>
   <property name="toolTip">
    <string>&amp;Bolas y varillas</string>
   </property>
  </action>
  <action name="actionCPKSpheres">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;CPK spheres</string>
   </property>
  </action>
  <action name="actionSticks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Sticks</string>
   </property>
  </action>
  <action name="actionWires">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wires</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
